### ISSUE TITLE
RRT: Request Approved Process

### ISSUE THREAD

**Issue by calexh-sar** [sartography/cr-connect#52]
After the PI has received an email notification of request approval, the they will need to sign in to RRP and complete the Check-on Request Status task.  Once this task is complete, the request status should be updated to "Open" and all Area Monitors associated with the PIs spaces notified as such.

**Comment by calexh-sar** [comment]
Won't do
